package/MDAnalysis/analysis/rdf.py

```diff
@@ -324,6 +351,7 @@ def _conclude(self):
                 N -= xA * xB * nblocks
 
             # Average number density
+            self.volume_cum = self.results.volume_cum
```

Comment:
my understanding is that you don't need to keep track of it (twice), since you're assigning it to `self.volume_cum` in `_conclude` anyway.